zerver/tests/test_narrow.py

```diff
@@ -694,9 +699,31 @@ def test_get_old_messages_with_search(self):
             meeting_message['match_content'],
             '<p>I am hungry!</p>')
 
+        narrow_emoji = [
+            dict(operator='sender', operand='cordelia@zulip.com'),
+            dict(operator='search', operand=':office:'),
+        ]
+        result_emoji = self.get_and_check_messages(dict(
+            narrow=ujson.dumps(narrow_emoji),
+            anchor=0,
+            num_after=10,
+        )) # type: Dict[str, Dict]
```

Comment:
Probably not even needed since the type of `result_emoji` can be determined from `get_and_check_message`.